scipy/stats/_continuous_distns.py

```diff
@@ -10286,6 +10286,140 @@ def _single_cdf(q, k, df):
                                           b=np.inf)
 
 
+class relativistic_bw_gen(rv_continuous):
+    r"""A relativistic Breit-Wigner random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    cauchy: Cauchy distribution, also known as the Breit-Wigner distribution.
+
+    Notes
+    -----
+
+    The probability density function for `relativistic_bw` is
+
+    .. math::
+
+        f(x, \rho) = \frac{k}{(x^2 - \rho^2)^2 + \rho^2}
+
+    where
+
+    .. math::
+        k = \frac{2\sqrt{2}\rho^2\sqrt{\rho^2 + 1}}
+            {\pi\sqrt{\rho^2 + \rho\sqrt{\rho^2 + 1}}}
+
+    The relativistic Breit-Wigner distribution is used in high energy
+    physics to model resonances. It gives the distribution for the observed
+    center-of-mass energy produced by a resonance given its mass :math:`M` and
+    decay-width :math:`\Gamma` [1]_, where :math:`M` and :math:`\Gamma` are
+    expressed in natural units. In SciPy's parametrization, the shape parameter
+    :math:`\rho` is equal to :math:`M/\Gamma`.
+
+    %(after_notes)s
+
+    :math:`\rho = M/\Gamma` and :math:`\Gamma` is the scale parameter. If one
+    seeks to model the :math:`Z^0` boson with :math:`M \approx 91.1876 text{
+    GeV}` and :math:`\Gamma \approx 2.4952\text{ GeV}` [2]_ one can set `rho`
+    equal to ``91.1876/2.4952`` and ``scale=2.4952``.
+
+    To ensure a physically meaningful result when using the `fit` method, one
+    should set ``floc=0`` to fix the location parameter to 0.
+
+    References
+    ----------
+    .. [1] Relativistic Breit-Wigner distriution, Wikipedia,
+           https://en.wikipedia.org/wiki/Relativistic_Breit-Wigner_distribution
+    .. [2] M. Tanabashi et al. (Particle Data Group) Phys. Rev. D 98, 030001 -
+           Published 17 August 2018
+
+    %(example)s
+
+    """
+    def _argcheck(self, rho):
+        return rho > 0
+
+    def _shape_info(self):
+        return [_ShapeInfo("rho", False, (0, np.inf), (False, False))]
+
+    def _pdf(self, x, rho):
```

Comment:
Would implementing `_logpdf`/`_logcdf` be useful? In that case, could we exponentiate them to implement the `pdf/cdf`?

Reply:
I’ll look into it.

Reply:
It could be left as an enhancement, too.